### DISCUSSION TITLE
Allowed clients

### DISCUSSION THREAD

**Discussion by kashikoy** [AdguardTeam/AdGuardHome#6888] [Q&A] [answered]
Hello,

Since the version 0.107.47, any client, even if not in "Allowed clients", can get a query answer.

Is there something new from 0.107.46 ?

Thank you.

Steven

**Comment by tescophil** [comment]
I've just raised a bug for this, my instance is answering external unauthorised DNS-over-TLS requests..

BIG security issue. not cool...

**Reply by ghost** [reply]
Hi @kashikoy,

Please follow along the issue [here](https://github.com/AdguardTeam/AdGuardHome/issues/6890) and provide information where you can.

**Comment by kashikoy** [comment, marked as answer]
Fix v0.107.48
